spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -80,32 +104,72 @@ definitions:
             type: CarrierPhase
             units: cycles
             desc: Carrier phase observation with typical sign convention.
+        - D:
+            type: Doppler
+            units: Hz
+            desc: Doppler observation with typical sign convention.
         - cn0:
             type: u8
-            units: dB Hz * 4
+            units: dB Hz * 0.25
             desc: Carrier-to-Noise density
         - lock:
-            type: u16
+            type: u8
             desc: |
-              Lock indicator. This value changes whenever a satellite
-              signal has lost and regained lock, indicating that the
-              carrier phase ambiguity may have changed.
+              Lock timer. This value gives an indication of the time
+              for which a signal has maintained continuous phase lock.
+              Whenever a signal has lost and regained lock, this 
+              value is reset to zero. It is encoded according to DF402 from
+              the RTCM 10403.2 Amendment 2 specification.  Valid values range 
+              from 0 to 15 and the most significant nibble is reserved for future use.
+        - flags:
+            type: u8
+            desc: |
+              Measurement status flags. A bit field of flags providing the
+              status of this observation.  If this field is 0 it means only the Cn0
+              estimate for the signal is valid.
+            fields:
+              - 3-7:
```

Comment:
@anth-cole ok I will add a detection to our received SBP processing that will mark zero cn0 as an invalid message instead of needed a flag.

@anth-cole hmm ok, although currently we don't "jump" carrier phase, only pseudorange? (which btw breaks RTCM/RINEX rules) see - https://github.com/swift-nav/exafore_planning/issues/419#issuecomment-256820434